### PR TITLE
pf9-comms dependency pkg added in pre-req checks

### PR DESCRIPTION
**Pull request by devidask27**
## ISSUE(S):

https://platform9.atlassian.net/browse/FT-352

## SUMMARY

pf9-comms installation was failing for one of the FT-user because logrotate was not installed on it.

## ISSUE TYPE

- [ ✓ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ x] Breaking change (fix or feature that may cause existing functionality to not work as expected)
- [ x] This change requires a documentation update

## IMPACTED FEATURES/COMPONENTS:

Impacted command
prep-node

#### Manual

**Prep-node is failing if logrotate is removed**
Screenshot 2022-03-01 at 12 47 35 PM
After adding logrotate as part of pre-req checks
Screenshot 2022-03-01 at 12 34 43 PM

#### Reviewers

/cc @anupbarve @nikhilbandi

### CODE REVIEW

File: pkg/platform/debian/debian.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ import (
 )
 
 var (
-	packages                   = []string{"curl", "uuid-runtime", "net-tools"}
+	packages                   = []string{"curl", "uuid-runtime", "net-tools", "logrotate"}
```
Comment (anupbarve) [inline pkg/platform/debian/debian.go:18]: shouldn't this be fixed by comms package installation itself ?

Comment (yogeshj83) [reply pkg/platform/debian/debian.go:18]: I believe, we have to decide what is a boundary for pf9ctl to do pre-install checks.

Comment (yogeshj83) [reply pkg/platform/debian/debian.go:18]: It was always confusing to me what part of pre-install checks falls under the purview of pf9ctl and which part falls under the purview of the host installer (host-agent, comms, etc.).

**Comment by devidask27** [comment]
Closing this one, since it is opened in core

**Pull request closed without merging**